src/style.css

```
@@ -1,3 +1,100 @@
+@font-face {
+  font-family: "Roboto";
+  src: url("fonts/Roboto-Medium.ttf") format("woff2");
+  font-weight: 500;
+  font-style: normal;
+}
+
+html {
+  --WHITE_COLOR: #fff;
+  --BLACK_COLOR: #000;
+  --AQUA_GRADIENT: #00acdc;
```

commonly css variables is used in lower case 
```suggestion
  --white_color: #fff;
  --black_color: #000;
  --aqua_gradient: #00acdc;
```